xpdan/data_reduction.py

```diff
@@ -103,21 +103,21 @@ def pull_dark(self, header):
 
     def _dark_sub(self, event, dark_img):
         """ priviate method operates on event level """
-        dark_sub_bool = False
+        dark_sub = False
         img = event['data'][self.image_field]
         if dark_img is not None and isinstance(dark_img, np.ndarray):
-            dark_sub_bool = True
+            dark_sub = True
             img -= dark_img
         ind = event['seq_num']
         event_timestamp = event['timestamps'][self.image_field]
-        return img, event_timestamp, ind, dark_sub_bool
+        return img, event_timestamp, ind, dark_sub
 
     def dark_sub(self, header):
         """ public method operates on header level """
         dark_img, dark_time_stamp = self.pull_dark(header)
         for ev in self.exp_db.get_events(header, fill=True):
-            sub_img, timestamp, ind, dark_sub_bool = self._dark_sub(ev,
-                                                                    dark_img)
+            sub_img, timestamp, ind, dark_sub = self._dark_sub(ev,
```

Comment:
We might want to not name a variable the same name as the function it is created in.